lib/topological_inventory/amazon/messaging_client.rb

```diff
@@ -0,0 +1,60 @@
+require "manageiq-messaging"
+
+module TopologicalInventory
+  module Amazon
+    class MessagingClient < TopologicalInventory::Providers::Common::MessagingClient
+      OPERATIONS_QUEUE_NAME = "platform.topological-inventory.operations-amazon".freeze
+      REFRESH_QUEUE_NAME = "platform.topological-inventory.collector-amazon".freeze
+
+      # Instance of messaging client for Worker
+      def worker_listener
+        @worker_listener ||= ManageIQ::Messaging::Client.open(worker_listener_opts)
+      end
+
+      def targeted_refresh_listener
+        @targeted_refresh_listener ||= ManageIQ::Messaging::Client.open(targeted_refresh_listener_opts)
+      end
+
+      def worker_listener_queue_opts
+        {
+          :auto_ack    => false,
+          :max_bytes   => 50_000,
+          :service     => OPERATIONS_QUEUE_NAME,
+          :persist_ref => "topological-inventory-operations-amazon"
+        }
+      end
+
+      def targeted_refresh_listener_queue_opts
```

Comment:
@pkomanek there is no targeted refresh except ansible tower, so remove everyting targeted_refresh related